### PR TITLE
Feature3/validations

### PR DESCRIPTION
**Pull request by Anand8317**
Added validations for the models
Added test for the models

### CODE REVIEW

File: spec/models/comment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```
Comment (FrederickMih) [inline spec/models/comment_spec.rb:3]: - [ ] Based on the requirements of this project, any custom method in models should have a corresponding unit test. Let's kindly test `#update_post_comment_counter`

File: spec/models/like_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
```
Comment (FrederickMih) [inline spec/models/like_spec.rb:3]: - [ ] We will equally required to lastly test the `#update_post_like_counter` method as pointed above.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: :raising_hand: @Anand8317, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- PR is professional :+1: 
- Unit tests are running with success :+1: 
- Great job understanding good testing approach :+1: 


## Required Changes ♻️
**README.md:** Hi @Anand8317, in the Contribution section of the readme file, the issue link is not working. I got 404 error page display when I click on the link. Let's kindly fix :wrench: the link for better user's experience

_Check the comments under the review._

## Optional suggestions
**N/A**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — COMMENTED** [review]
Hi @Anand8317,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly ensure all your readme header links are linked to the required sections. Most of them are broken. For better professionalism, we should avoid broken links. If you have no sections for some links then kindly remove them, like live demo, etc.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by DammyShittu — APPROVED** [review]
Hi @Anand8317,

## APPROVED 🟢

## Highlights
✔️ Custom methods tests

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**